src/service_manager.rs

```diff
@@ -208,4 +208,28 @@ impl ServiceManager {
             Ok(Service::new(unsafe { ScHandle::new(service_handle) }))
         }
     }
+
+    /// Retrieves the service name of the specified service.
```

Comment:
I know this is just a verbatim copy of the MS docs.. But I think we can do better. I personally find it very hard to know when and how to use this from just reading this docs and seeing the function signature. See `create_service` and `open_service` above for inspiration. Please write more detailed docs for this method.

Reply:
Done